applications/asset_tracker/src/env_sensors/CMakeLists.txt

```diff
@@ -5,4 +5,31 @@
 #
 
 zephyr_include_directories(.)
-target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/env_sensors.c)
+
+if (CONFIG_USE_BME680_BSEC)
+	if(CONFIG_FP_HARDABI)
+		target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/bsec.c)
+		set (bsec_dir ${NRF_DIR}/ext/BSEC_1.4.7.4_Generic_Release)
```

Comment:
The version should be in it's own variable, possibly even in a Kconfig.

Reply:
The problem is that BSEC_1.4.7.4 is not drop in compatible with BSEC_1.4.7.3. This "forces" the user to use 1.4.7.4 unless they are really intersted in changing it.

Reply:
What about when there is a version 1.4.7.5 ?

Reply:
Added kconfig